### ISSUE TITLE
[Schema Inaccuracy/Question] Can `owner` be null in `repo-search-result-item` component

### ISSUE THREAD

**Issue by gr2m** [github/rest-api-description#549]
# Schema Inaccuracy

```yml
    repo-search-result-item:
      title: Repo Search Result Item
      description: Repo Search Result Item
      type: object
      properties:
        id:
          type: integer
        node_id:
          type: string
        # ...
        owner:
          "$ref": "#/components/schemas/nullable-simple-user"
```

## Expected

See https://github.com/octokit/types.ts/issues/338#issuecomment-918706205. We'd expect that `owner` cannot be set to null. And if it can, could you describe the use case in the property description?

## Reproduction Steps

n/a

**Comment by nickfloyd** [comment]
Please let us know if this work is still needed.  We're trying to follow up on any older issues to make sure they get properly routed.

**Comment by creativecreatorormaybenot** [comment]
@nickfloyd, if you read through the few messages in my issue in https://github.com/octokit/types.ts/issues/338#issue-935227682, you will see that this is still an unresolved discussion, and since it is unexpected, we are missing the explanation for why the `owner` can be null (which is this issue here).

TL;DR: this has not yet been addressed and remains valid.
